runbotics/runbotics-desktop/src/action/sharepoint/excel/types.ts

```diff
@@ -3,25 +3,20 @@ import { DesktopRunRequest } from 'runbotics-sdk';
 export type FileActionRequest =
 | DesktopRunRequest<'sharepointExcel.setCell', SharepointSetExcelCellActionInput>
 | DesktopRunRequest<'sharepointExcel.getCell', SharepointGetExcelCellActionInput>
-| DesktopRunRequest<'sharepointExcel.openFileFromSite', SharepointOpenFromSiteActionInput>
-| DesktopRunRequest<'sharepointExcel.openFileFromRoot', SharepointOpenFromRootActionInput>
 | DesktopRunRequest<'sharepointExcel.closeSession', SharepointExcelCloseSessionActionInput>
 | DesktopRunRequest<'sharepointExcel.updateRange', SharepointExcelUpdateRangeActionInput>
-| DesktopRunRequest<'sharepointExcel.getRange', SharepointExcelGetRangeActionInput>;
+| DesktopRunRequest<'sharepointExcel.getRange', SharepointExcelGetRangeActionInput>
+| DesktopRunRequest<'sharepointExcel.openFile', SharepointExcelOpenFileActionInput>;
 
-
-export type SharepointOpenFromRootActionInput = {
-    filePath: string;
-    worksheetName: string;
+export interface SharepointExcelOpenFileActionInput {
+    platform:string;
```

Comment:
why `platform: string` if you know what we expect `SharePoint | OneDrive` a then you don't need to do `@ts-ignore`